ubi/src/os.rs

```diff
@@ -19,7 +19,7 @@ pub(crate) fn linux_re() -> &'static Lazy<Regex> {
 }
 
 pub(crate) fn macos_re() -> &'static Lazy<Regex> {
-    regex!(r"(?i:(?:\b|_)(?:darwin|mac(?:os)?|osx)(?:\b|_))")
+    regex!(r"(?i:(?:\b|_)(?:darwin|mac(?:os|osx)?|osx)(?:\b|_))")
```

Comment:
I ran into the same issue today and wanted to open a PR, but it was already open :-). What do you think about simplifying the regex to this?

```suggestion
    regex!(r"(?i:(?:\b|_)(?:darwin|mac(?:osx?)?|osx)(?:\b|_))")
```

Reply:
I like your version, it looks really simpler 👍